src/client/Microsoft.Identity.Client/Platforms/Features/WinFormsLegacyWebUi/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -209,11 +209,10 @@ private bool CheckForClosingUrl(Uri url, byte[] postData = null)
                 readyToClose = true;
             }
 
-            if (!readyToClose && !url.Scheme.Equals("https", StringComparison.OrdinalIgnoreCase) &&
-                !url.AbsoluteUri.Equals("about:blank", StringComparison.OrdinalIgnoreCase) && !url.Scheme.Equals("javascript", StringComparison.OrdinalIgnoreCase))
+            if (!readyToClose && EmbeddedUiCommon.IsAllowedIeOrEdgeAuthorizationUri(url)) // IE error pages                
```

Comment:
I think this should be `!`EmbeddedUiCommon.IsAllowedIeOrEdgeAuthorizationUri(url) because the logic was changed from NOT ANDs to ORs?

Reply:
Yes, should have tested that.